ipamd/ipamd.go

```diff
@@ -149,15 +148,14 @@ var (
 
 // IPAMContext contains node level control information
 type IPAMContext struct {
-	awsClient     awsutils.APIs
-	dataStore     *datastore.DataStore
-	k8sClient     k8sapi.K8SAPIs
-	eniConfig     eniconfig.ENIConfig
-	dockerClient  docker.APIs
-	networkClient networkutils.NetworkAPIs
-
-	currentMaxAddrsPerENI int
-	maxAddrsPerENI        int
+	awsClient        awsutils.APIs
+	dataStore        *datastore.DataStore
+	k8sClient        k8sapi.K8SAPIs
+	customNetworkCfg bool
+	eniConfig        eniconfig.ENIConfig
+	dockerClient     docker.APIs
+	networkClient    networkutils.NetworkAPIs
+	maxAddrsPerENI   int
```

Comment:
I would rename this to maxIPsPerENI to be consistent with the functions and the previous var name.